docker-compose.yml

```diff
@@ -33,9 +33,7 @@ volumes:
     external: false
   osfstoragecache_vol:
     external: false
-  osf_requirements_vol:
-    external: false
-  osf_requirements_3_10_vol:
+  osf_requirements_3_12_vol:
```

Comment:
Since we did this, I think we can remove all
```yaml
# - osf_requirements_vol:/usr/local/lib/python3.6
```
in this file. A total of 7 occurrences if my count is correct.